Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/StructurePanel.java

```diff
@@ -1114,4 +1122,114 @@ private IncludedStructuralElement getTreeNodeStructuralElement(TreeNode treeNode
         }
         return null;
     }
+
+    /**
+     * Get List of MediaUnits assigned to multiple IncludedStructuralElements.
+     *
+     * @return value of severalAssignments
+     */
+    List<MediaUnit> getSeveralAssignments() {
+        return severalAssignments;
+    }
+
+
+    /**
+     * Get the index of this StructureTreeNode's MediaUnit out of all MediaUnits
+     * which are assigned to more than one IncludedStructuralElement.
+     *
+     * @param treeNode object to find the index for
+     * @return index of the StructureTreeNode's MediaUnit if present in the List of several assignments, or -1 if not present in the list.
+     */
+    public int getMultipleAssignmentsIndex(StructureTreeNode treeNode) {
+        if (Objects.nonNull(treeNode.getDataObject()) && treeNode.getDataObject() instanceof View
+                && Objects.nonNull(((View) treeNode.getDataObject()).getMediaUnit())) {
+            return severalAssignments.indexOf(((View) treeNode.getDataObject()).getMediaUnit());
+        }
+        return -1;
+    }
+
+    /**
+     * Check if the selected Node's MediaUnit is assigned to several IncludedStructuralElements.
+     *
+     * @return {@code true} when the MediaUnit is assigned to more than one logical element
+     */
+    public boolean isAssignedSeveralTimes() {
+        if (Objects.nonNull(selectedLogicalNode) && selectedLogicalNode.getData() instanceof  StructureTreeNode) {
+            StructureTreeNode structureTreeNode = (StructureTreeNode) selectedLogicalNode.getData();
+            if (Objects.nonNull(structureTreeNode.getDataObject()) && structureTreeNode.getDataObject() instanceof View) {
```

Comment:
```suggestion
            if (structureTreeNode.getDataObject() instanceof View) {
```
instanceof already checks against null